.github/workflows/static_checks.yml

```diff
@@ -107,3 +107,7 @@ jobs:
       - name: Regex Patterns Validation Check
         run: |
           bazel run //scripts:pattern_validation_check -- $(pwd) app data utility domain
+
+      - name: XML Syntax Validation Check
+        run: |
+          bazel run //scripts:xml_syntax_check -- $(pwd) app data utility domain
```

Comment:
data domain doesn't have any res file, utility does file dimen and string XML file if we are checking that else no needed, we can keep the check only for app module.

Reply:
Yes! they don't have any res files, but my only concern was that these layers do have the `AndroidManifest.xml` file present, so I thought of including them too in the XML syntax check. 
WDYT?

Reply:
Looking for @BenHenning thoughts on this

Reply:
Hmm. I just left a comment in #3340 about passing in the root directory and just recursing among that. If we validate all XML files then this slight oddity is less noticeable (since we do ultimately want all XML files to be valid). Would that mitigate the concern here?

Reply:
Done.